features/cli/oc_import_image.feature

```diff
@@ -108,13 +108,13 @@ Feature: oc import-image related feature
     When I run the :get client command with:
       | resource        | imagestreams |
     Then the output should match:
-      | .*deployment-example.* |
+      | .*deployment-example.* | 
     When I run the :get client command with:
       | resource_name   | deployment-example |
       | resource        | dc                 |
       | o               | yaml               |
     Then the output should match:
-      |[Ll]astTriggeredImage.*deployment-example@sha256.*|
+      | .*openshift/deployment-example@sha256.* |
```

Comment:
`.*` means anything.  Perhaps we can just use `openshift/deployment-example@sha256` ?

Reply:
Yeah, done!